index.html

```diff
@@ -6,14 +6,64 @@
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css\style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+<body class="body">
+    <header class="header">
+     <nav class="navigation">
+        <img class="logo" src="img\karma-logo.svg" alt="Karma Logo">
+        <label class="toggle">&#9776;</label>
+        <input type="checkbox" id="toggle">
+        
```

Comment:
Nice work with the hamburger menu on smaller screens 🙂 At the moment though when I click on it I'm not getting any links